hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/EventRegistrationTest.java

```diff
@@ -55,6 +56,8 @@ public class EventRegistrationTest extends HazelcastTestSupport {
 
     @Rule
     public final OverridePropertyRule overridePropertyRule = set(HAZELCAST_TEST_USE_NETWORK, "true");
+    @Rule
+    public final OverridePropertyRule overridePropertyRule2 = set(WAIT_SECONDS_BEFORE_JOIN_ASYNC.getName(), "false");
```

Comment:
why is this property override required here?

Reply:
This tests batching of requests. Batching will no longer happen in the async strategy (will always be false) thus the test will fail.
In other words, the test is only applicable in sync repartition strategy.